sources/packages/forms/src/form-definitions/sfaa2025-26.json

```diff
@@ -11080,7 +10917,8 @@
             "eq": ""
           },
           "properties": {},
-          "lockKey": true
+          "lockKey": true,
+          "customConditional": "show = data.dependantStatus === \"independant\";"
```

Comment:
It is my understanding that this should not be displayed for Part-time applications but apparently it is. Can you please verify?

Reply:
Changed the to only show BC residency to either part-time or full-time when all the fields are populated and student considered independant. Thanks!